pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -530,6 +731,9 @@ func TestJwtFilter(t *testing.T) {
 
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
+			defaultValue := features.EnableRemoteJwks
```

Comment:
this should be enabled only for the specific test cases otherwise it means we only tested the new code path.

Reply:
This tests both the flows because the defaultValue is false and only the new tests will set `enableRemoteJwks` to true.